File: BHAVRP_Java/BHAVRPv2.0/BHAVRP2.0/src/cujae/inf/ic/om/heuristic/assignment/AssignmentTemplate.java
package cujae.inf.ic.om.heuristic.assignment;

import cujae.inf.ic.om.problem.output.solution.Solution;

public abstract class AssignmentTemplate implements IAssignment{
	
	public void initialize() {};
	public void assign() {};
	public Solution finish() {
		return null;
	}
}

File: BHAVRP_Python/cujae/inf/citi/om/assignment/assignment_template.py
from .i_assignment import IAssignment
from ..problem.solution.solution import Solution

class AssignmentTemplate(IAssignment):
    
    def __init__(self):
        super().__init__()
    
    def initialize(self):
        pass
    
    def assign(self):
        pass
    
    def finish(self) -> Solution:
        return None